conf/waivers-released

```diff
@@ -2,12 +2,12 @@
 # which are not available in their final form in the Anaconda environment
 # - see https://github.com/ComplianceAsCode/content/issues/9746
 /hardening/anaconda(/with-gui)?/[^/]+/firewalld_sshd_port_enabled
-    rhel >= 8
-# similar case for RHEL 9
 # https://github.com/ComplianceAsCode/content/issues/10939
-/hardening/anaconda(/with-gui)?/[^/]+/firewalld_loopback_traffic_restricted
-/hardening/anaconda(/with-gui)?/[^/]+/firewalld_loopback_traffic_trusted
-    rhel >= 9
+# https://github.com/ComplianceAsCode/content/issues/10613
+/hardening/anaconda(/with-gui)?/[^/]+/firewalld_loopback_traffic_(restricted|trusted)
+# https://github.com/ComplianceAsCode/content/issues/11564
+/hardening/image-builder(/with-gui)?/[^/]+/firewalld_loopback_traffic_(restricted|trusted)
+    rhel >= 8
```

Comment:
Do we want to mix IB here? Or should the IB waiver be among other IB waivers?

It's about adding rule to "rule specific section" vs "test specific section". This one you added close to existing Anaconda waiver (as it's the same rule), but for `firewalld_sshd_port_enabled` you added it to IB waivers section instead of to Anaconda waiver. Let's make it consistent.

Reply:
Hmm, good point.

I used a new section only for IB-specific fails, anything shared with other tests should probably be combined with the existing waivers. I'll move `firewalld_sshd_port_enabled` to the anaconda part.

Btw, I'll remove `firewalld_sshd_port_enabled` waivers completely (incl. Anaconda) from `waivers-upstream` and re-test everything, because I'm pretty sure the remediation for that rule has been removed, since we don't need an IB waiver either.